pwm/pwm.sls

```diff
@@ -0,0 +1,108 @@
+#installs packages needed for PWM, downloads the WAR file and installs it into tomcat
```

Comment:
The name of this sls is confusing. When the formula is installed, the sls reference ends up being `pwm.pwm`, which doesn't really provide any context. Maybe `install.sls` would be a better filename, since it is installing the necessary packages, services, and the war file?

Reply:
changed it to intall_pwm.sls